### PR TITLE
2D Animations

### PR DESCRIPTION
**Pull request by Santhosh-M31**

### CODE REVIEW

File: advanced-qgis.Rmd [hunk 1]
```diff
@@ -1,702 +1,736 @@
...
+```{r echo=FALSE, fig.align='center', out.width='75%'}
+knitr::include_graphics('images/advanced_qgis/timeseries8.png')
+```
+
+9. Click **Insert an Expression** button and enter the following expression to display the year.
+
+```{r echo=FALSE, fig.align='center', out.width='75%'}
+knitr::include_graphics('images/advanced_qgis/timeseries9.png')
+```
+``format_date(@map_start_time, 'yyyy-MM-dd')``
```
Comment (spatialthoughts) [inline advanced-qgis.Rmd:562]: expression comes before image. Also use triple-backquotes

File: advanced-qgis.Rmd [hunk 2]
```diff
@@ -1,702 +1,736 @@
...
+
+# 2D Animations
+
+Time is an important component of many spatial datasets. Along with location information, time providers another dimension for analysis and visualization of data. If you are working with dataset that contains timestamps or have observations recorded at multiple time-steps, you can easily visualize it using the **TimeManager** plugin in QGIS 3.14 or above.
+
+## Exercise: Create a GIF showing changes in piracy hotspots over time
+
+We will continue to work with the maritime piracy dataset. First, we will create a heatmap visualization and then animate the heatmap to show how the piracy hot-spots have changed over the past 2 decades.
+
+1. Open the **maritime_piracy** project from the data package. There are thousands of incidents and it is difficult to determine with more piracy. Rather than individual points, a better way to visualize this data is through a heatmap. Select the ASAM_events layers and click the **Open the layer Styling Panel** button in the Layers panel. Click the **Single symbol** drop-down.
```
Comment (spatialthoughts) [inline advanced-qgis.Rmd:509]: ASAM_events has to be in backquotes

File: advanced-qgis.Rmd [hunk 3]
```diff
@@ -1,702 +1,736 @@
...
+
+We will continue to work with the maritime piracy dataset. First, we will create a heatmap visualization and then animate the heatmap to show how the piracy hot-spots have changed over the past 2 decades.
+
+1. Open the **maritime_piracy** project from the data package. There are thousands of incidents and it is difficult to determine with more piracy. Rather than individual points, a better way to visualize this data is through a heatmap. Select the ASAM_events layers and click the **Open the layer Styling Panel** button in the Layers panel. Click the **Single symbol** drop-down. 
+
+```{r echo=FALSE, fig.align='center', out.width='75%'}
+knitr::include_graphics('images/advanced_qgis/timeseries1.png')
+```
+
+2. In the renderer selection drop-down, select **Heatmap** renderer. Next, select the Viridis color ramp from the Color ramp selector. Adjust the Radius value to **5.0**. At the bottom, expand the Layer Rendering section and adjust the Opacity to **75.0%**. This gives a nice visual effect of the hotspots with the land layer below.
```
Comment (spatialthoughts) [inline advanced-qgis.Rmd:515]: Layer rendering has to be in * * for italics.

**Review by spatialthoughts — COMMENTED** [review]
Overall ok. You have to fix all the steps to be consistent with formatting that we follow. Most of them are wrong, so review step-by-step.

Layer names in backquootes (`layer`)
GUI labels in Italics (*ilaitcs*)
Selections in bold (**bold**)

Left a few comments.

**Comment by Santhosh-M31** [comment]
17a0d46 is final with the comments updated. Please check

File: advanced-qgis.Rmd [hunk 4]
```diff
@@ -1,702 +1,736 @@
...
+
+We will continue to work with the maritime piracy dataset. First, we will create a heatmap visualization and then animate the heatmap to show how the piracy hot-spots have changed over the past 2 decades.
+
+1. Open the **maritime_piracy** project from the data package. There are thousands of incidents and it is difficult to determine with more piracy. Rather than individual points, a better way to visualize this data is through a heatmap. Select the ASAM_events layers and click the **Open the layer Styling Panel** button in the Layers panel. Click the **Single symbol** drop-down. 
+
+```{r echo=FALSE, fig.align='center', out.width='75%'}
+knitr::include_graphics('images/advanced_qgis/timeseries1.png')
+```
+
+2. In the renderer selection drop-down, select **Heatmap** renderer. Next, select the **Viridis** color ramp from the Color ramp selector. Adjust the Radius value to **5.0**. At the bottom, expand the Layer Rendering section and adjust the Opacity to **75.0%**. This gives a nice visual effect of the hotspots with the land layer below.
```
Comment (spatialthoughts) [inline advanced-qgis.Rmd:515]: Layer Rendering in italics

File: advanced-qgis.Rmd (hunk 2 above)
Comment (spatialthoughts) [reply advanced-qgis.Rmd:509]: Not yet done.

**Review by spatialthoughts — COMMENTED** [review]
Review each step thoroughly again.

**Pull request merged**